### PR TITLE
update to  20w17a

### PR DESCRIPTION
**Pull request by sbh1311**
if I managed to create the new packet and placed it in the correct place I think this all is good
tell me if there things need to be changed

### CODE REVIEW

**Comment by sbh1311** [comment]
I used 1.15.2 files for this because I forgot what I changed and what not if you like I can use the previous snapshot and send a pull request again

**Comment by rom1504** [comment]
Yes it would be better, no point in redoing the work (and checking) of what was done in previous snapshot

**Comment by sbh1311** [comment]
hi I changed it sorry it took time I didn't have time the last few days

File: data/pc/20w17a/protocol.json [hunk 1]
```diff
@@ -4990,37 +5003,39 @@
...
-                    "0x25": "update_command_block_minecart",
-                    "0x26": "set_creative_slot",
-                    "0x27": "update_jigsaw_block",
-                    "0x28": "update_structure_block",
-                    "0x29": "update_sign",
-                    "0x2a": "arm_animation",
-                    "0x2b": "spectate",
-                    "0x2c": "block_place",
-                    "0x2d": "use_item"
+                    "0x0f": "generate_structure",
```
Comment (rom1504) [inline data/pc/20w17a/protocol.json:5006]: you need to add generate_structure in above mapping as well

**Comment by rom1504** [comment]
cannot be merged without fixes

**Comment by sbh1311** [comment]
hi sorry i have no time to finish it so I will close it for now 
sorry for the wast of time ~.~

**Pull request closed without merging**